app/functions.js

```diff
@@ -0,0 +1,59 @@
+
+const { MajorColorNames, MinorColorNames, ColorPair } = require('./colorsDS');
+

```

Comment:
The filename `functions.js` is too generic. It doesn't say anything about the responsibilities implemented in this file.
It also doesn't give guidance to future developers who want to add functionality. Any function they write, can it come and sit in this file?

Reply:
Yes , I will give appropriate name to it. 
In my opinion only functionality related to color conversion must be present  so I will name it colorConversion or I will think of something better.